pages/linux/aplay.md

```diff
@@ -0,0 +1,15 @@
+# aplay
+
+> Command-line sound player for ALSA soundcard driver.
+
+- Play file "foobar". Sampling rate, bit depth, and so forth will be automatically determined for supported soundfile formats:
```

Comment:
I'm not sure if the clarification is needed, because is just play a file without tweaks. Also, is determined by the file format or the file metadata?
```suggestion
- Play a specific file (sampling rate, bit depth, etc. will be automatically determined for the file format):
```